library/Fission/App/Creator/Class.hs

```diff
@@ -29,17 +29,25 @@ type Errors = OpenUnion
    ]
 
 class Monad m => Creator m where
-  create :: UserId -> CID -> UTCTime -> m (Either Errors (AppId, Subdomain))
+  create :: 
+       UserId 
+    -> CID 
+    -> Natural 
+    -> UTCTime 
+    -> m (Either Errors (AppId, Subdomain))
 
 instance (MonadIO m, App.Domain.Initializer m) => Creator (Transaction m) where
-  create ownerId cid now = do
+  create ownerId cid size now = do
     appId <- insert App
       { appOwnerId    = ownerId
       , appCid        = cid
+      , appSize       = size
       , appInsertedAt = now
       , appModifiedAt = now
       }
 
+    _ <- insert $ CreateAppEvent appId ownerId cid size now
```

Comment:
we weren't inserting `CreateAppEvent`, so I added that in
I also added `appId` to the model